native-image/what-the-dickens/lab/pom.xml

```diff
@@ -9,12 +9,12 @@
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
     <groupId>com.example</groupId>
-    <artifactId>What_the_Dickens</artifactId>
+    <artifactId>what_the_dickens</artifactId>
```

Comment:
I changed the `artifactId` for lowercase.  Otherwise, Paketo Buildpacks build fails with:
```
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:3.0.5:build-image (default-cli) on project What_the_Dickens: Execution default-cli of goal org.springframework.boot:spring-boot-maven-plugin:3.0.5:build-image failed: Unable to parse name "What_the_Dickens". Image name must be in the form '[domainHost:port/][path/]name', with 'path' and 'name' containing only [a-z0-9][.][_][-] -> [Help 1]
```